contribute-to-tidb/write-document.md

```diff
@@ -1,3 +1,229 @@
 # Write Document
 
 Good documentation is crucial for any kind of software. This is especially true for sophisticated software systems such as distributed database like TiDB. The TiDB community aims to provide concise, precise, and complete documentation and welcomes any contribution to improve TiDB's documentation.
+
+## What you can contribute
+
+You can start from any one of the following items to help improve TiDB Docs at the PingCAP website ([en](https://docs.pingcap.com/tidb/stable) and [zh](https://docs.pingcap.com/zh/tidb/stable)):
+
+- Fix typos or format (punctuation, space, indentation, code block, etc.)
+- Fix or update inappropriate or outdated descriptions
+- Add missing content (sentence, paragraph, or a new document)
+- Translate docs changes from English to Chinese, or from Chinese to English
+- Submit, reply to, and resolve [docs issues](https://github.com/pingcap/docs/issues)
+- (Advanced) Review Pull Requests created by others
+
+## Before you contribute
+
+Before you contribute, please take a quick look at some general information about TiDB documentation maintenance. This can help you to become a contributor soon.
```

Comment:
For "some general information", do you mean the content below? It seems some information outside here to me.

Reply:
"let's take a quick look.." would reduce the confusion over "please take a quick look..."

Reply:
```suggestion
Before you contribute, let's take a quick look at some general information about TiDB documentation maintenance. This can help you to become a contributor soon.
```